python/lsst/ap/association/diaForcedSource.py

```diff
@@ -264,3 +266,31 @@ def _calibrate_and_merge(self,
         output_catalog.drop(columns=self.config.dropColumns, inplace=True)
 
         return output_catalog
+
+    def _trim_to_bbox(self, catalog, exposure):
```

Comment:
It seems a bit odd to have a `trim_to_bbox` method that takes an exposure. Why not let it take a `Box2I` (more reusable) or call it `trim_to_exposure` (more robust to changes in algorithm, pixel vs. celestial coordinates, etc)?

Reply:
Changed to `trim_to_exposure`